applications/FluidDynamicsApplication/python_scripts/trilinos_navier_stokes_two_fluids_solver.py

```diff
@@ -144,18 +144,25 @@ def _CreateLevelSetConvectionProcess(self):
         linear_solver = self._GetLinearSolver()
         computing_model_part = self.GetComputingModelPart()
         epetra_communicator = self._GetEpetraCommunicator()
+
+        levelset_convection_settings = self.settings["levelset_convection_settings"]
+
+        levelset_convection_settings["levelset_variable_name"].SetString("DISTANCE")
+        levelset_convection_settings["levelset_convection_variable_name"].SetString("VELOCITY")
+        levelset_convection_settings["levelset_gradient_variable_name"].SetString("DISTANCE_GRADIENT")
+
```

Comment:
```suggestion
```
Here it is an example of why defaults should be in the json string to avoid potential issues (aside of the duplication in the derived solver).